geode-core/src/main/java/org/apache/geode/internal/cache/CachePerfStats.java

```diff
@@ -1310,10 +1323,6 @@ public void incEntryCount(int delta) {
     stats.incLong(entryCountId, delta);
   }
 
-  public long getEntries() {
-    return stats.getLong(entryCountId);
-  }
-
```

Comment:
This seems fine, but I'm just curious what the reasoning was for removing this. Was there a problem with the way it was retrieving the entry count from the stats?

Reply:
In general, we're hoping to discourage using stats for anything other than publishing metrics. Removing the getter helps with that. In this case, we had an additional (minor) concern that someone might call this method expecting it to have an up-to-date value. If the `CachePerfStats` is actually a `RegionPerfStats`, this stat's value will lag, because the value is refreshed only periodically during sampling.